Mathlib/NumberTheory/NumberField/Units/Basic.lean

```diff
@@ -135,8 +135,11 @@ instance : Nonempty (torsion K) := One.instNonempty
 /-- The torsion subgroup is cyclic. -/
 instance [NumberField K] : IsCyclic (torsion K) := subgroup_units_cyclic _
 
-/-- The order of the torsion subgroup as a positive integer. -/
-def torsionOrder [NumberField K] : ℕ+ := ⟨Fintype.card (torsion K), Fintype.card_pos⟩
+/-- The order of the torsion subgroup. -/
+abbrev torsionOrder [NumberField K] : ℕ := Fintype.card (torsion K)
```

Comment:
IMHO, `abbrev`s should be used very rarely. Why not just add a `NeZero` instance? BTW, can `deriving` find it?